### PR TITLE
Review 2

### PR DESCRIPTION
**Pull request by duythien**
Implement case unit test file conten render suport prefix and suffix

### CODE REVIEW

File: lib/Drupal/at_base/Tests/Unit/ServiceContentRenderTest.php [hunk 1]
```diff
@@ -74,4 +74,21 @@ public function testDynamicVariables() {
     $data['controller'] = array('At_Base_Test_Class', 'hi');
     $this->assertEqual($expected, $this->render->render($data));
   }
+
+  /**
+   * Conten render Support prefix, suffix
+   * @return html
+   */
+  public function testRenderPrefix() {
```
Comment (and1truong) [inline lib/Drupal/at_base/Tests/Unit/ServiceContentRenderTest.php]: Rename testPrefixSuffix().

File: lib/Drupal/at_base/Tests/Unit/ServiceContentRenderTest.php [hunk 2]
```diff
@@ -74,4 +74,21 @@ public function testDynamicVariables() {
...
+    $data = array();
+    $data['template'] = '@atest_base/templates/block/render_template.html.twig';
+    $data['variables'] = array(
+        'name' => 'Drupal',
+        'prefix' => '<div id="abc">',
+        'suffix' => '</div>'
+    );
```
Comment (and1truong) [inline lib/Drupal/at_base/Tests/Unit/ServiceContentRenderTest.php:88]: We are not planning support this style. But:

```
$data = array(
  'content' => 'anything', // also support 'function' => 'my_callback', …
  'prefix' => '<div>',
  'suffix' => '</div>',
);
```

File: tests/atest_base/templates/block/render_template.html.twig [hunk 3]
```diff
@@ -0,0 +1 @@
+{{ prefix ~ name ~ suffix }}
```
Comment (and1truong) [inline tests/atest_base/templates/block/render_template.html.twig:1]: Prefix/Suffix must support all render types, not just template.

File: tests/atest_config/config/config.yml [hunk 4]
```diff
@@ -1,3 +1,3 @@
 foo: bar
 structured_data:
-  foo: bar
+  foo: helllo
```
Comment (and1truong) [inline tests/atest_config/config/config.yml:3]: Do not this change.

File: at_base.module [hunk 5]
```diff
@@ -8,6 +8,9 @@
  *
  *  - Remove at_fn(), at_fn_fake()
  *  - Update docs for service container > reuse option.
+ *
+ *  - Test github command stash
+ *  - Balala
```
Comment (and1truong) [inline at_base.module]: Remove these two lines.

File: lib/Drupal/at_base/Tests/Unit/ServiceContentRenderTest.php (hunk 2 above)
Comment (duythien) [inline lib/Drupal/at_base/Tests/Unit/ServiceContentRenderTest.php:88]: It Working

File: lib/Drupal/at_base/Tests/Unit/ServiceContentRenderTest.php [hunk 6]
```diff
@@ -74,4 +74,21 @@ public function testDynamicVariables() {
     $data['controller'] = array('At_Base_Test_Class', 'hi');
     $this->assertEqual($expected, $this->render->render($data));
   }
+
+  /**
+   * Conten render Support prefix, suffix
+   * @return html
```
Comment (and1truong) [inline lib/Drupal/at_base/Tests/Unit/ServiceContentRenderTest.php]: Must method return nothing.

File: lib/Helper/Content_Render.php [hunk 7]
```diff
@@ -66,6 +66,8 @@ public function render($data = NULL) {
       $this->setdata($data);
     }
 
+    var_dump($this->build());
```
Comment (and1truong) [inline lib/Helper/Content_Render.php]: VERY NICE (:

**Comment by and1truong** [comment]
Fixed in https://github.com/andytruong/render/issues/9

**Pull request closed without merging**